lib/OpenQA/WebAPI/Plugin/Helpers.pm

```diff
@@ -376,6 +383,15 @@ sub register {
 
     $app->helper(compose_job_overview_search_args => \&_compose_job_overview_search_args);
     $app->helper(param_hash                       => \&_param_hash);
+    $app->helper(
+        link_key_exists => sub {
+            my ($c, $value) = @_;
+            my @link_keys = split ",", $c->app->config->{job_settings_ui}->{keys_to_render_as_links};
+            if (@link_keys) {
+                return 1 if grep /$value/, @link_keys;
+            }
```

Comment:
The `grep` is bad because it would match also, e.g. `SCHEDULE` if `keys_to_render_as_links` contains e.g. `YAML_SCHEDULE`. Since this example is not hypothetical it would be good to fix this.

And when doing that anyways: Don't split the string from the config in each table row. I'd rather do the splitting one time when the configuration has been parsed. And it would also make sense to use a hash here and check for the existence of hash keys. That way the lookup should be faster than with grep and you also get the exact matching.